src/docker/composebuilder/service_builder_test.go

```diff
@@ -15,28 +15,29 @@ var _ = Describe("ComposeBuilder", func() {
 	var _ = Describe("building external dependencies", func() {
 		var dockerCompose *types.DockerCompose
 		var serviceEndpoints map[string]*endpoints.ServiceEndpoint
+		var serviceRole string
```

Comment:
I'd prefer not having this variable and instead harding the service role everywhere. I think that makes the test easier to understand. Duplication in tests doesn't matter as much to me and I would rather have it be easy to understand